qemu/tests/cfg/numa_cpu.cfg

```diff
@@ -0,0 +1,81 @@
+- numa_cpu:
+    type = numa_cpu
+    kill_vm = yes
+    no Host_RHEL.m6, Host_RHEL.m7
+    # '-numa cpu' was added from QEMU 3.0(av-8.0),rhel-8.2 don't support it.
+    # 'dies' was added from QEMU 4.1(av-8.1, rhel-8.3), rhel-8.2 don't support.
+    # libvirt still does not support it by 8.3, so test it from 8.3 here
+    no Host_RHEL.m8.u0, Host_RHEL.m8.u1, Host_RHEL.m8.u2
```

Comment:
How about from 4.1 ?  Both '-numa cpu' and 'dies' are supported from then on.
I am thinking that, since libvirt still have not supported '-numa cpu' by now, it is not too risky.

Reply:
Okay for me.